### PR TITLE
chore: add basic table

### PR DESCRIPTION
**Pull request by mhaack**
Relates #336 

Test URLs:
- Before: https://main--hlx-test--urfuwo.hlx.live/blog/the-c-suite-gets-an-ai-upgrade
- After: https://basic-table--hlx-test--urfuwo.hlx.live/blog/the-c-suite-gets-an-ai-upgrade

- [x] New Blocks introduced in this PR

### CODE REVIEW

**Review by saurabh-khare — APPROVED** [review]

File: aemedge/blocks/table/table.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ */
+
+function buildCell(rowIndex) {
+  const cell = rowIndex ? document.createElement('td') : document.createElement('th');
+  if (!rowIndex) cell.setAttribute('scope', 'col');
+  return cell;
+}
+
+export default async function decorate(block) {
+  const table = document.createElement('table');
```
Comment (saurabh-khare) [inline aemedge/blocks/table/table.js:14]: dom-builder.js can be used

File: aemedge/blocks/table/table.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/*
+ * Table Block
```
Comment (saurabh-khare) [inline aemedge/blocks/table/table.js:2]: Comments

File: aemedge/blocks/table/table.js (hunk 1 above)
Comment (mhaack) [reply aemedge/blocks/table/table.js]: Fixed, took that over from boilerplate project.

File: aemedge/blocks/table/table.js (hunk 2 above)
Comment (mhaack) [reply aemedge/blocks/table/table.js]: Fixed, took that over from boilerplate project.

**Pull request merged**